components/carbon-transports/websocket/org.wso2.carbon.websocket.transport/src/main/java/org/wso2/carbon/websocket/transport/WebsocketConnectionFactory.java

```diff
@@ -235,6 +236,19 @@ public void operationComplete(ChannelFuture channelFuture) throws Exception {
         return null;
     }
 
+    private static String deriveSubprotocol(String wsSubprotocol, String contentType){
+        String subprotocol = null;
+        if (wsSubprotocol != null) {
+            subprotocol = wsSubprotocol;
```

Comment:
we can directly return from here.

Reply:
private static String deriveSubprotocol(String wsSubprotocol, String contentType) {
        if (wsSubprotocol != null) {
            return wsSubprotocol;
        }
        return wsSubprotocol != null ? SubprotocolBuilderUtil.contentTypeToSyanapeSubprotocol(contentType) : null;
    }